tools/lsp/util.rs

```diff
@@ -149,6 +149,16 @@ pub fn with_property_lookup_ctx<R>(
         .map(|n| object_tree::type_from_node(n, &mut Default::default(), tr))
         .or_else(|| scope.last().map(|e| e.borrow().lookup_property(prop_name).property_type));
 
+    // try to match properties from `PropertyAnimation`
+    if ty.is_none() || ty == Some(Type::Invalid) {
+        match prop_name {
+            "delay" | "duration" => ty = Some(Type::Duration),
+            "iteration-count" => ty = Some(Type::Float32),
+            "easing" => ty = Some(Type::Easing),
+            _ => (),
+        }
+    }
```

Comment:
The reason why I added this condition check is because unlike `element.PropertyDeclaration()` which is able to call `.Type()` to get type info. There's no way to parse type info for properties inside of `animate` block, but type info here is vital to be assigned to `lookup_context.property_type` and latter will used when completing `easing` property.